pkg/services/libraryelements/database.go

```diff
@@ -198,8 +199,10 @@ func (l *LibraryElementService) deleteLibraryElement(c context.Context, signedIn
 			return ErrLibraryElementNotFound
 		}
 
+		elementID = element.ID
 		return nil
 	})
+	return elementID, err
```

Comment:
Maybe add some tests for this change?
https://github.com/grafana/grafana/blob/main/pkg/services/libraryelements/libraryelements_delete_test.go

Reply:
Done!

Reply:
Thank you!